kornia/enhance/equalization.py

```diff
@@ -241,27 +230,27 @@ def _compute_equalized_tiles(interp_tiles: torch.Tensor, luts: torch.Tensor) ->
 
     # internal regions
     tl, tr, bl, br = preinterp_tiles_equalized[:, 1:-1, 1:-1].unbind(3)
-    t = tiw * (tl - tr) + tr
-    b = tiw * (bl - br) + br
-    tiles_equalized[:, 1:-1, 1:-1] = tih * (t - b) + b
+    t = torch.addcmul(tr, tiw, torch.sub(tl, tr))
+    b = torch.addcmul(br, tiw, torch.sub(bl, br))
+    tiles_equalized[:, 1:-1, 1:-1] = torch.addcmul(b, tih, torch.sub(t, b))
 
     # corner regions
     tiles_equalized[:, 0::gh - 1, 0::gw - 1] = preinterp_tiles_equalized[:, 0::gh - 1, 0::gw - 1, 0]
 
     # border region (h)
     t, b, _, _ = preinterp_tiles_equalized[:, 1:-1, 0].unbind(2)
-    tiles_equalized[:, 1:-1, 0] = tih.squeeze(1) * (t - b) + b
+    tiles_equalized[:, 1:-1, 0] = torch.addcmul(b, tih.squeeze(1), torch.sub(t, b))
```

Comment:
Is addcmul faster ?

Reply:
Yes, because the same aten function is running both ops. At least with nsight I saw a noticeable change.

Reply:
I tried to use JIT to see if all the ops could be joined and computed together in the same cuda thread but it was a waste of time :(. If we have an expert in optimisation at Kornia we could discuss a little bit about these things. :)

Reply:
@t-vi any insights?